addons/web_editor/static/src/components/media_dialog/file_selector.js

```diff
@@ -157,7 +159,36 @@ export class FileSelector extends Component {
             this.state.attachments = await this.fetchAttachments(this.NUMBER_OF_ATTACHMENTS_TO_DISPLAY, 0);
         });
 
-        this.debouncedScroll = useDebounced(this.scrollToLoadMoreButton, 500);
+        this.debouncedOnScroll = useDebounced(this.updateScroll, 15);
+        this.debouncedScrollUpdate = useDebounced(this.updateScroll, 500);
+
+        useEffect(
+            (modalEl) => {
+                if (modalEl) {
+                    modalEl.addEventListener("scroll", this.debouncedOnScroll);
+                    // Positionning the bottom bar right above the footer.
+                    const footerEl = modalEl.parentElement.querySelector("footer");
+                    const {top, width} = footerEl.getBoundingClientRect();
+                    this.loadMoreButtonRef.el.style.setProperty("--footer-top", `${top}px`);
+                    this.loadMoreButtonRef.el.style.setProperty("--footer-width", `${width}px`);
+                    return () => {
+                        modalEl.removeEventListener("scroll", this.debouncedOnScroll);
+                    };
```

Comment:
To discuss tomorrow, can't this be simplified without the need of the footer if we go with absolute bottom and extract the show more icon ?
Unless I am missing a big thing, it's too complicated to be good (I mean there is surely an easier way)